openshift_scalability/config/router-arp_stress_README.md

```diff
@@ -0,0 +1,19 @@
+# README for router-arp_stress.yaml configuration
+
+[router-arp_stress.yaml](https://github.com/openshift/svt/blob/master/openshift_scalability/config/router-arp_stress.yaml) is a [cluster_loader](https://github.com/openshift/svt/tree/master/openshift_scalability) configuration which can be used to test if a cluster has issues with [incorrect ARP cache tuning for large clusters](https://docs.openshift.com/container-platform/3.4/install_config/router/default_haproxy_router.html#deploy-router-arp-cach-tuning-for-large-scale-clusters)
+
+**Note:**  This test is very resource intensive can cause cluster failures.  It must only be used in test environments.
+
+Example of running the test:
+
+The environment for this test is a cluster on AWS EC2 with 1 master/etcd node, 1 router node and 6 application nodes.   All instances are m4.xlarge.
+
+1. Clone [cluster_loader](https://github.com/openshift/svt/tree/master/openshift_scalability)
+2. Edit svt/openshift_scalability/config/router-arp_stress.yaml and change the number of projects to 12
```

Comment:
I'd leave 2. out.  Just merge https://github.com/openshift/svt/pull/169.  1100 is enough (1100 > 1024).